frontend/src/assets/i18n/fr.json

```diff
@@ -562,6 +562,11 @@
             },
             "comparison": {
                 "legend-desc": "L'image montre la différence relative d'impact cumulé entre le scénario de référence et le scénario anticipé."
+            },
+            "overflow": {
+                "title": "Dépassement de la valeur maximale",
+                "detail": "Veuillez noter que la partie du résultat cumulé qui dépasse la valeur maximale par pixel n'est pas reflétée dans le rapport. Le résultat pour les bandes affectées est inférieur à ce qui serait attendu si l'ensemble du changement paramétré était pris en compte, sans limiter la valeur au maximum.",
+                "list-description": "La valeur maximale a été dépassée dans les bandes suivantes:"
```

Comment:
@bastienqueste It would be greatly appreciated if you could review these translations 🙏

Reply:
Let's put a pin in this. I'm merging this suggestion as is, for the time being.